src/main/java/dev/code/digital_lending_microservice/payload/request/CreateCustomerDto.java

```diff
@@ -0,0 +1,17 @@
+package dev.code.digital_lending_microservice.payload.request;
+
+import lombok.Data;
+
+@Data
+public class CreateCustomerDto {
+
+    private String firstName;
+
+    private String lastName;
+
+    private String phoneNumber;
```

Comment:
the database stores 11 digits ensure that characters are not more than 11.

Reply:
ok